### ISSUE TITLE
Export the Filtered Data

### ISSUE THREAD

**Issue by asjones987** [deldersveld/rDataTable#4]
Would be cool to export the filtered data to Excel or CSV.

**Comment by ZedLeb** [comment]
Yes I second this.  Such functionality exists in the PBI table visual (annoyingly so sometimes - if one has forgotten that a filter is applied!)  

That's why I like R Data Tables filters - you have a visual indication of  applied filters - which is not easily the case with a sliced table.

**Comment by kcinmelbourne** [comment]
Agreed - love the ability to search on every column but then to not have the ability to export based on those filters is a big restriction for my stakeholders.  Thanks in advance.
